waddrmgr/manager.go

```diff
@@ -82,7 +82,7 @@ const (
 
 	// NumInitialAddrs is the number of internal and external addresses to
 	// pre-generate when the wallet is created.
-	NumInitialAddrs = 20
+	NumInitialAddrs = 10
```

Comment:
Why was this reduced?

Reply:
I picked that number at random. Thought it was more significant than it was.